crates/linker/src/lib.rs

```diff
@@ -92,13 +93,19 @@ pub fn build_and_preprocess_host(
     )
 }
 
+fn metadata_file_name(target: &Triple) -> String {
+    let target_triple_str = get_target_triple_str(target);
+
+    format!("metadata_{}.rm1", target_triple_str.unwrap_or("unknown"))
```

Comment:
I think the file extension should be sufficient for us to know it's a metadata file!
```suggestion
    format!("{}.rm1", target_triple_str.unwrap_or("unknown"))
```

Reply:
It is sufficient for those who already know the extension but isn't it nicer for newcomers to immediately know the purpose of a file? Otherwise, every newcomer would need to ask us or go digging in the code base.